packages/synapse-react-client/src/components/HasAccess/HasAccessV2.integration.test.tsx

```diff
@@ -260,12 +298,12 @@ describe('HasAccess tests', () => {
 
       renderComponent(props, wrapperProps)
 
-      const icon = await screen.findByRole('img', {
-        name: 'You must request access to this restricted file.',
-      })
-      expect(icon.getAttribute('data-svg')).toEqual('accessClosed')
+      await expectIcon(
+        'accessClosed',
+        'You must request access to this restricted file.',
+      )
       // User will be prompted to sign in in the AR modal
-      expect(screen.queryByRole('button')).not.toBeInTheDocument()
+      await expectArButton('Request Access')
```

Comment:
Previously used a button to open sign in modal and a link to open the AR modal, but now using a button in both cases so must check for the type of button rather than asserting that there are no buttons in the document